packages/preview/ieee-monolith/0.1.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Taiki M.
```

Comment:
I'm no license expert, but because you based your work on charged-ieee, I feel like the original authors should be credited here too.

Reply:
No, I just assumed charged-ieee was under MIT, so it's fine to not give attribution here.